simple_minimization.py

```diff
@@ -0,0 +1,63 @@
+from model_training import Data, Model
+from scipy.optimize import minimize
+import numpy as np
+from time import time
+try:
+    from joblib import Parallel, delayed
+    joblib_import = True
+except ImportError:
+    joblib_import = False
+
+class Minimizer:
+    def __init__(self, flux, model, p0=(5777, 4.44, 0.00)):
+        self.flux = flux
+        self.model = model
+        self.p0 = p0
+
+    def minimize(self, max_iter=1000, tol=0.01):

```

Comment:
Add different methods of minimization routines here which are already [implemented in scipy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html).